### DISCUSSION TITLE
Welcome!

### DISCUSSION THREAD

**Discussion by kokonut27** [PyScript-Language/PyScript-Compiler#6] [General]
## Welcome!

Here you can talk about the language, questions about it, and more.

If you want to join, just ask me, or set up an issue, discussion, or PR, and I'll get back to you with info and more! 
However, I might decline, so please don't feel down about it.
Thanks! :)

If you want to, you can set up your own discussion, but please follow the [`Code of Conduct`](https://github.com/PyScript-Language/PyScript-Compiler/blob/ce88fe84e3ef32ab3db83b706a8b5afc48b2f838/CODE_OF_CONDUCT.md) and the [`GitHub Guidelines`](https://docs.github.com/articles/github-community-guidelines).
Thanks again!

That's really it. Just follow the instructions and rules, and you should be fine!
Bai!

**Comment by kokonut27** [comment]
Editing go brrrr

**Reply by iop3** [reply]
first person to comment (after you xD)

**Reply by kokonut27** [reply]
lol xD

**Reply by iop3** [reply]
xD
